### ISSUE TITLE
Latest RELEASE file

### ISSUE THREAD

**Issue by gregaubert** [ArekSredzki/electron-release-server#80]
Hi!
Is there a way to get the latest RELEASE file ? A route like `http://download.myapp.com/update/win32/latest/RELEASES` ?

Edit:
Well I just noticed that whatever the existing version I use I always get the same RELEASE file...
I don't really see the point of having to put the version in the url since it always return the last file ?
Am I doing something wrong ?

**Comment by ArekSredzki** [comment]
The concept of a "latest" `RELEASES` file doesn't really make sense because of how it is used.
The reason why you must provide the version that you are requesting the file from (which is quite trivial anyway) is to prevent downgrading the application.

The `RELEASES` file will contain the latest version that *has an update compatible asset available* for it if the version requesting it is `<=` the candidate version. If the RELEASES version has the same file then the MD5 hash is used to ensure that the version is, in fact, the same (that's just how squirrel for windows works).

TL;DR, just use the current version in the request.
